group/assignment6/RearrangingCars.java

```diff
@@ -0,0 +1,75 @@
+import java.util.ArrayList;
+import java.util.LinkedList;
+import java.util.Queue;
+
+public class RearrangingCars {
+
+    public static ArrayList<Car> reArrangeCars(ArrayList<Car> cars){
```

Comment:
The assigment says that the method should return a sequence of moves. I see that you print these moves. Perhaps you can change your code, so that it will return the moves instead of printing them.

Reply:
Yes, I realized this after I uploaded the solution. Will do, thanks a lot ! :)